src/pages/signup/index.js

```diff
@@ -7,12 +7,10 @@ import { localize, WithIntl } from 'components/localization'
 import { SEO } from 'components/containers'
 import device from 'themes/device.js'
 import { Header, Text } from 'components/elements'
-import Award from 'images/svg/award-winning.svg'
-import Complaint from 'images/svg/complaint.svg'
-import Security from 'images/svg/security.svg'
+import Graph from 'images/svg/graph.svg'
 
 const Wrapper = styled.section`
-    padding-top: 8rem;
+    padding: 8rem 0;
     width: 100%;
     height: inherit;
```

Comment:
instead, change this height to calc(100vh - 10.4rem)

10.4rem is the header height

Reply:
ignore this, make the main doing this instead